docs/developer-docs/backend/rust/index.md

```diff
@@ -1,15 +1,28 @@
 # Introduction to Developing Canisters in Rust
 
-Rust is a powerful and type-sound modern programming language with an active developer community. Because Rust compiles to WebAssembly, it offers a rich development environment for writing dapps to run on the Internet Computer blockchain. To help pave the way for writing dapps in Rust that can be deployed on the Internet Computer blockchain, DFINITY provides the [Rust canister development kit (Rust CDK)](https://github.com/dfinity/cdk-rs) to simplify the process.
+Rust is a powerful and type-sound modern programming language with an active developer community. Because Rust compiles to WebAssembly, it offers a rich development environment for writing dapps to run on the Internet Computer blockchain. 
 
-The CDK consists of the following crates:
+# Getting Started
+To help pave the way for writing dapps in Rust that can be deployed on the Internet Computer blockchain, you can use the [IC SDK](../../setup/install/index.mdx). The IC SDK supports the Rust as well as the Motoko programming language. To create a Rust project using the IC SDK, all one needs to do is add the `--type=rust` flag while creating a new project. For example, here we create a Rust project named `hello_world`:
+
+```bash
+dfx new --type=rust hello_world
+```
+
+To start a new project see [Rust Quickstart](./rust-quickstart.md).
+
+## Under the Hood
+
+To support Rust development, the IC SDK uses cargo crates which make up the [Rust canister development kit (Rust CDK)](https://github.com/dfinity/cdk-rs). 
```

Comment:
```suggestion
To support Rust development, the IC SDK includes the [Rust canister development kit (Rust CDK)](https://github.com/dfinity/cdk-rs). 
```

Reply:
here is how I would word it. I think the part about cargo crates is not very relevant to the reader.